src/cmd/manage_nsfs.js

```diff
@@ -289,6 +290,7 @@ async function add_bucket(data) {
     const exists = await native_fs_utils.config_file_exists(fs_context, bucket_conf_path);
     if (exists) throw_cli_error(ManageCLIError.BucketAlreadyExists, data.name.unwrap());
 
+    extract_properties_from_bucket_data_and_validate_schema(data);
```

Comment:
@shirady 
A few days ago I noticed that when creating an account while specifying a distinguished name instead of uid and gid, we accidentally assign false to uid and gid instead of a number and this would break the schema, can you please check that?

Reply:
@romayalon I checked it.
- I will suggest a small fix in this PR (separate commit for code review).
- It doesn't break the schema (if it was I had to fix it before...).

To reproduce it, we can add this print:
`console.log('SDSD res', res);`
in the test (it) `cli account create `under (describe) `cli account flow distinguished_name - happy path`.
Before fix:

> [LOG] CONSOLE:: SDSD res {
>   "response": {
>     "code": "AccountCreated",
>     "reply": {
> ...
>       "nsfs_account_config": {
>         "distinguished_name": "moti1003",
>         "uid": false,
>         "gid": false,
>         "new_buckets_path": "/private/tmp/test_bucketspace_fs/root_path_manage_nsfs/new_buckets_path_user2/"
> ...

After fix:

> [LOG] CONSOLE:: SDSD res {
>   "response": {
>     "code": "AccountCreated",
>     "reply": {
> ...
>       "nsfs_account_config": {
>         "distinguished_name": "moti1003",
>         "new_buckets_path": "/private/tmp/test_bucketspace_fs/root_path_manage_nsfs/new_buckets_path_user2/"
> ...

Reply:
setting a property to false (type boolean) instead of a number must break the schema.
I wonder if you didn't see it because of the oneOf keyword added to the schema.
please add additionalProperties: false to the schema (both account and bucket), I suppose that after adding it you should see the error.

Reply:
@romayalon I added it both in the account and bucket schema as the last line and rerun the mentioned test above (using the original code before the change) and still didn't get an error from the schema.
After setting the `additionalProperties: false` to each list of the `oneOf` and re-running the test again I received 3 failing tests including the one mentioned above), therefore in nsfs_account_schema you'll see the `additionalProperties: false` appears 3 times.

Reply:
I checked what we are doing in our existing schema code and it looks like we use schema_utils.strictify(schema, options, base). This should set additionalProperties: false recursively and do some more stuff that we might want to do

Reply:
I added it (`schema_utils.strictify`), thanks.